Automattic/wp-calypso
# Gutenboarding: Extract DomainPickerButton Component

#### Changes proposed in this Pull Request

Extract `DomainPickerButton` component from header.
We'll be adding some content to it that depends on a bunch of data that we don't wanna mix into the header component.

#### Testing instructions

Add http://calypso.localhost:3000/gutenboarding, verify that the domain picker still works as before.
